gaia-9000/genesis.json

```diff
@@ -0,0 +1,7649 @@
+            }
+        ]
+    },
+    {
+        "address": "cosmos12fm2n7pkmtqk7untyhs9my749jyzef3vs49kha",
+        "coins": [
+            {
+                "denom": "steak",
+                "amount": "50"
+            }
+        ]
+    },
+    {
+        "address": "cosmos12285eep5w4jdv50qfhwqxem9duyhkvyav2ff7l",
+        "coins": [
+            {
+                "denom": "steak",
+                "amount": "50"
+            }
+        ]
+    },
+    {
+        "address": "cosmos12vs0pqsg5y8pzvq5ztdzzmgjxh2ucxsqqvu8v7",
+        "coins": [
+            {
+                "denom": "steak",
+                "amount": "50"
+            }
+        ]
+    },
+    {
+        "address": "cosmos12wfc2qvlqnedmtqcstsanx245qlyppdhnzare4",
+        "coins": [
+            {
+                "denom": "steak",
+                "amount": "50"
+            }
+        ]
+    },
+    {
+        "address": "cosmos12wnqvqg79s9jqrul0lva3h72rfmewm7jx9ltzf",
+        "coins": [
+            {
+                "denom": "steak",
+                "amount": "50"
+            }
+        ]
+    },
+    {
+        "address": "cosmos120skmenn2a0ra8y6zassrxvnfc5rlme8yxps0d",
+        "coins": [
+            {
+                "denom": "steak",
+                "amount": "50"
+            }
+        ]
+    },
+    {
+        "address": "cosmos120uqj330zdue5yynulh2a4h5gyj2tcayzh4d4r",
+        "coins": [
+            {
+                "denom": "steak",
+                "amount": "50"
+            }
+        ]
+    },
+    {
+        "address": "cosmos123ahlc3vkfe8hf354gdcddr3lzxdmyg2rm3d99",
+        "coins": [
+            {
+                "denom": "steak",
+                "amount": "50"
+            }
+        ]
+    },
+    {
+        "address": "cosmos12hqns2dc9zuzmyukehmvvvhzymyudgrxxjsfkj",
+        "coins": [
+            {
+                "denom": "steak",
+                "amount": "50"
+            }
+        ]
+    },
+    {
+        "address": "cosmos12ceualfg92x7du73mtcv0zya4nxvq3tldagell",
+        "coins": [
+            {
+                "denom": "steak",
+                "amount": "50"
+            }
+        ]
+    },
+    {
+        "address": "cosmos126ayk3hse5zvk9gxfmpsjr9565ef72pvzwku8m",
+        "coins": [
+            {
+                "denom": "steak",
+                "amount": "50"
+            }
+        ]
+    },
+    {
+        "address": "cosmos12mj0nqeuk698n8drdtkumvpuxmtfylkqfgzzl2",
+        "coins": [
+            {
+                "denom": "steak",
+                "amount": "50"
+            }
+        ]
+    },
+    {
+        "address": "cosmos12uggfcy84ghj4sepeyd95ym2wahw2l73w3yplv",
+        "coins": [
+            {
+                "denom": "steak",
+                "amount": "50"
+            }
+        ]
+    },
+    {
+        "address": "cosmos12u2e2qn74zlk34g223l7jthx53t2wgcj8g5xjx",
+        "coins": [
+            {
+                "denom": "steak",
+                "amount": "50"
+           
```

Comment:
@cwgoes Do this look like a correct way to configure proposer rewards?
I'm not clear on how proposer rewards and overall inflation are related

Reply:
you have currently set this to 10000% hahaha - 

The default values are here:
https://github.com/cosmos/cosmos-sdk/blob/bfc7944b55382c0749fecf294289e3b65792c4dc/x/distribution/types/genesis.go#L42

for your understanding `sdk.NewDecWithPrec(1, 2)` is equivalent to `0.01` which is equivalent to saying "hay that value is 1%"